packages/sdk-components-react/src/form.ws.tsx

```diff
@@ -1,11 +1,11 @@
 import { FormIcon } from "@webstudio-is/icons/svg";
-import { form } from "../css/normalize";
 import {
   defaultStates,
   type PresetStyle,
   type WsComponentMeta,
   type WsComponentPropsMeta,
-} from "./component-meta";
+} from "@webstudio-is/react-sdk";
+import { form } from "@webstudio-is/react-sdk/css-normalize";
```

Comment:
are we going to keep react-sdk? I thought we will move react-sdk stuff that is framework agnostic to "sdk" or is this a separate step

Reply:
Yep, here only components split